src/advanced_agent/database/models.py

```
@@ -0,0 +1,275 @@
+
+class Conversation(Base):
+    """会話記録テーブル"""
+    __tablename__ = 'conversations'
+    
+    id = Column(String(36), primary_key=True, default=lambda: str(uuid.uuid4()))
+    session_id = Column(String(36), ForeignKey('agent_states.session_id'), nullable=False)
+    user_input = Column(Text, nullable=False)
+    agent_response = Column(Text, nullable=False)
+    timestamp = Column(DateTime, nullable=False, default=datetime.utcnow)
+    importance_score = Column(Float, nullable=False, default=0.5)
+    response_time = Column(Float, nullable=True)  # 応答時間（秒）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)